docs/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,101 @@
+# Deploy Worker and UDF Binaries General Instruction
+
+This documentation will describe how to deploy Worker and UDF (User Defined Function) binaries, which Environment Variable to set up and some commonly used options when launching applications with `spark-submit`.
+
+## Table of Contents
```

Comment:
This is a pretty concise guide, do we need a table of contents?

Reply:
Yep, I was also thinking about these. I think table of contents will be useful if we have more FAQs in the future. But I am happy to remove it for now. Thanks